webapp/src/containers/WalletPage/components/WalletTxnsNew/index.tsx

```diff
@@ -296,6 +296,12 @@ const WalletTxns: React.FunctionComponent<WalletTxnsProps> = (
     if (type === POOL_SWAP_CATEGORY_LABEL) {
       return <MdCompareArrows className={styles.typeIcon} />;
     }
+    if ([REMOVE_LIQUIDITY_LABEL].includes(type)) {
+      return <MdArrowUpward className={styles.typeIcon} />;
```

Comment:
REMOVE_LIQUIDITY_LABEL is already defined above. I think we need to clarify this ticket

Reply:
update